### PR TITLE
Login and register pages need routing

### PR DESCRIPTION
**Pull request by sinha24**
### Changes in this PR:
In assignments>>F-02-93
- Login component
- Register component

### Reference GitHub issues:
[UI/UX] Create login page #16

***Routing is remaining in pages, most probably js/app.js needs some edits

### CODE REVIEW

File: assignments/F-02-93/js/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
```
Comment (Jo-ravar) [inline assignments/F-02-93/js/components/Login.js]: move this code in `miniproject/pages/index.js` .

File: assignments/F-02-93/js/components/Register.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
```
Comment (Jo-ravar) [inline assignments/F-02-93/js/components/Register.js]: Move code in `miniproject/pages/register.js`

**Review by Jo-ravar — CHANGES_REQUESTED** [review]
Move code to miniproject folder within pages .We are using next.js and not webpack here

File: assignments/F-02-93/js/components/Login.js (hunk 1 above)
Comment (sinha24) [reply assignments/F-02-93/js/components/Login.js]: Done

File: assignments/F-02-93/js/components/Register.js (hunk 2 above)
Comment (sinha24) [reply assignments/F-02-93/js/components/Register.js]: Done

**Comment by sinha24** [comment]
Routing still remaining

**Review by Jo-ravar — APPROVED** [review]
Login and register Page are very basic and will need enhancement

**Pull request merged**